keyboards/tweetydabird/lotus58/elite_c/info.json

```diff
@@ -1,3 +1,40 @@
 {
-    "bootloader": "atmel-dfu"
+    "pin_compatible": "elite_c",
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "F5", "pin_b": "F4", "resolution": 2}
+        ]
+    },
+    "matrix_pins": {
+        "cols": ["B1", "B2", "B3", "B6", "F7", "F6"],
+        "rows": ["D4", "C6", "D7", "E6", "B4"]
+    },
+    "split": {
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {"pin_a": "F4", "pin_b": "F5", "resolution": 2}
+                ]
+            }
+        },
+        "handedness": {
+            "pin": "B5"
+        },
+        "soft_serial_pin": "D2",
+        "transport": {
+            "protocol": "serial",
+            "sync": {
+                "indicators": true,
+                "layer_state": true,
+                "matrix_state": true,
+                "modifiers": true,
+                "oled": true
+            }
+        },
```

Comment:
Looks like this all already exists at the keyboard level.
```suggestion
```

Reply:
The serial protocol needs to be there. The rest I agree with